addons/ballistics/CfgMagazines.hpp

```diff
@@ -3,7 +3,99 @@ class CfgMagazines {
 
     class CA_Magazine;
     class VehicleMagazine;
-    
+
```

Comment:
bom?

Reply:
honestly not sure

Reply:
bom in middle of file doesn't make sense :u